rest-server/src/config/launcher.js

```diff
@@ -30,6 +30,7 @@ const logger = require('./logger');
 // get config from environment variables
 let launcherConfig = {
   hdfsUri: process.env.HDFS_URI,
+  webhdfsUri: process.env.WEBHDFS_URI,
```

Comment:
do we need webhdfs? native hdfs is enough.
if you need to use webhdfs, pls update deployment configuration accordingly.

Reply:
WehHDFS is more direct and convenient in this scenario, since it does not rely on calling an external local executable nor require us to do extra work to parse the result in plain text.